txp/Pos/Script/Examples.hs

```diff
@@ -80,6 +81,9 @@ intValidator = fromE $ parseValidator [text|
             False -> failure } }
     |]
 
+intValidatorWithVersionOne :: Script
+intValidatorWithVersionOne = setVersionOne intValidator
+
```

Comment:
@gromakovsky is this what you had in mind ?

Reply:
Yes.